vsintegration/src/FSharp.Editor/Common/Extensions.fs

```diff
@@ -163,6 +163,8 @@ module private SourceText =
 
                   member _.CopyTo(sourceIndex, destination, destinationIndex, count) =
                       sourceText.CopyTo(sourceIndex, destination, destinationIndex, count)
+
+                  member _.GetSubTextFromRange(range) = failwith "Not implemented"
```

Comment:
Actually, it would be quite useful to have this in the Editor, I think the very same implementation could work. Could you propagate it here?

Reply:
Probably same can be done for the implementation in the benchmarking.